<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderListItemWidget</class>
 <widget class="QWidget" name="OrderListItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>70</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MemberItemWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MemberItemWidget{background-color: rgb(255, 255, 255);};</string>
  </property>
  <widget class="QWidget" name="FilletWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>780</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>730</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>780</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#FilletWidget{background:rgb(252,252,252);border-radius:4px;border:1px solid rgb(235,235,235);}
QWidget#FilletWidget:hover,pressed{background:rgb(246,251,255);border-radius:4px;border:1px solid rgb(187,223,255);};</string>
   </property>
   <widget class="QLabel" name="labelCreateTime">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
background-color: rgba(255, 255, 255, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelState">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>178</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:px solid rgba(223,70,100,1);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrderCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrderName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
background-color: rgba(255, 255, 255, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelStateColor">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>25</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>178</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:5px solid rgba(223,70,100,1);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDetial">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
background-color: rgba(255, 255, 255, 0);
border:0px;color:blue</string>
    </property>
    <property name="text">
     <string>详情</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAction">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
background-color: rgba(255, 255, 255, 0);
border:0px;color:blue</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
